tiledb/sm/rest/rest_client.cc

```diff
@@ -30,6 +30,15 @@
  * This file declares a REST client class.
  */
 
+// clang-format off
+#ifdef TILEDB_SERIALIZATION
+#include "tiledb/sm/serialization/capnp_utils.h"
+#include "tiledb/sm/serialization/query.h"
```

Comment:
These items were not previously subject to TILEDB_SERIALIZATION, would any build configurations lacking TILEDB_SERIALIZATION still need them?
rest_client.cc must have been building before without capnp (TILEDB_SERIALIZATION) itself being active for build, as rest_client.cc appears to have always been built not subject to that flag...

Reply:
All the actual definitions in `rest_client.cc` are under `TILEDB_SERIALIZATION` so I think this is ok.

Reply:
Ok.